terraform/neutron.tf

```diff
@@ -73,6 +73,7 @@ resource "openstack_networking_router_v2" "router_mgmt" {
   name                    = "${var.prefix}-rtr"
   description             = "router for mgmtcluster (managed by terraform)"
   external_network_id     = data.openstack_networking_network_v2.external.id
+  # comment this out if your cloud does not have network AZs
```

Comment:
*terraform fmt* doesn't like it
```diff
$ terraform fmt -check -diff terraform/neutron.tf 
terraform/neutron.tf
--- old/terraform/neutron.tf
+++ new/terraform/neutron.tf
@@ -70,9 +70,9 @@
 }
 
 resource "openstack_networking_router_v2" "router_mgmt" {
-  name                    = "${var.prefix}-rtr"
-  description             = "router for mgmtcluster (managed by terraform)"
-  external_network_id     = data.openstack_networking_network_v2.external.id
+  name                = "${var.prefix}-rtr"
+  description         = "router for mgmtcluster (managed by terraform)"
+  external_network_id = data.openstack_networking_network_v2.external.id
   # comment this out if your cloud does not have network AZs
   availability_zone_hints = [var.availability_zone]
 }
```

Reply:
Maybe we can place it next to availability_zone.

Reply:
> Maybe we can place it next to availability_zone.

Looks like this is the only solution to keep the indentation.
But the line is quite long then.
I'm not sure what is "better practice", spontaneous I would prefer the "wrong" indentation by `terraform fmt` over a long line.

Reply:
@garloff @chess-knight after some more research I saw Gruntwork recommends [120 column limit](https://docs.gruntwork.io/guides/style/terraform-style-guide/#120-column-limit) in which we still are, if we place the comment next to the attribute.
That would change my mind and I would go with it. Some opinions?

Reply:
I am OK with that, also when I look on the terraform [style conventions](https://developer.hashicorp.com/terraform/language/syntax/style) I can see e.g. `count = 2 # meta-argument first` line there.

Reply:
Done